README.md

```diff
@@ -219,14 +219,15 @@ The following options are available.
  * `--venv`: folder in which to place the virtual env for tests (will be shared by all tests), overrides `INMANTA_TEST_ENV`.
    This options depends on symlink support. This does not work on all windows versions. On windows 10 you need to run pytest in an
    admin shell. Using a fixed virtual environment can speed up running the tests.
+ * `--pip-index-url`: pip index to install dependencies from. Can be specified multiple times to add multiple indexes. When set, it will overwrite the system index-url even if `pip-use-system-config` is set. (overrides `PIP_INDEX_URL`, defaults to `[]`)
+ * `--pip-pre`, `--no-pip-pre` Allow installation of pre-release package by pip or not? (overrides `PIP_PRE`, defaults to  `--install-mode != release` )
```

Comment:
Unsure, but this is more-or-less backward compatible

Reply:
I would like to get rid of it, if we can. It's a big if. I think most Jenkinsfiles set `PIP_PRE` anyway, so it could be worth a shot? But perhaps merge it in a second stage (meaning we shouldn't release pytest-inmanta yet) to isolate fallout?